client/src/modules/stock/inventories/registry.js

```diff
@@ -212,6 +212,9 @@ function StockInventoriesController(
     vm.hasError = false;
     vm.loading = true;
 
+    if (filters.purchase_uuid) {
+      filters.includeEmptyLot = 1;
```

Comment:
Can we just send this from the client side instead of putting it on the server?

Reply:
okay